packages/react/src/components/Tree/TreeItem.tsx

```diff
@@ -64,8 +64,8 @@ export interface TreeItemProps extends UIComponentProps, ChildrenComponentProps
   /** Called when the item's parent is about to be focused. */
   onFocusParent?: ComponentEventHandler<TreeItemProps>
 
-  /** Whether or not the item is in the open state. Only makes sense if item has children items. */
-  open?: boolean
+  /** Whether or not the item is in the expanded state. Only makes sense if item has children items. */
+  expanded?: boolean
```

Comment:
This is a breaking change, please add entry in the changelog under this section.